### ISSUE TITLE
Batch Normalization explained (YouTube)

### ISSUE THREAD

**Issue by chippen001** [chippen001/Capstone-Resources#15]
### Title
Batch Normalization- Part 2: How it works & Essence of Beta & Gamma

### URL
https://www.youtube.com/watch?v=2xChdY2qkmc

### Summary 
This video highlights the calculations done in the batch normalization process (normalizing/scale & shift) and why they are helpful in machine learning models. Normalization vs standardization is explained.

### Key Points 
-Normalization: collapse inputs to between 0 and 1.
-Standardization: mean 0, variance 1.
-Internal Covariate Shift: shift in distribution of hidden layer features during training. Use BN to solve this.
-Process:

-Normalize: x-hat = (x-mu)/stdv or (x-mu)/sqrt(variance + e). Mu is mean of x in mini-batch, stdv is standard deviation of x in mini-batch, e is epsilon (to avoid division by 0). 
-Scale and Shift: x-bar = gamma*x-hat + beta
- increases learning capability (parameters trying to learn overall distribution of training dataset).
- beta/gamma updated for every iteration to fit a global view

-BN comes before sigmoid function (activation function)

### Citation
ML for Nerds. Batch normalization- part 2: How it works essence of beta & gamma, 2023.
